cmd/scriggo/generate_go1.23_test.go

```diff
@@ -0,0 +1,98 @@
+//go:build go1.23
```

Comment:
Since [Reimplement checking generic and general declarations](https://github.com/open2b/scriggo/pull/951/commits/bed823174f8424f05c030089c372f07209948be3), this file has been removed, and its content were merged into the file `generate_test.go`, which already has a minimum Go version 1.22.